package.json

```diff
@@ -17,22 +17,28 @@
     "> 1%"
   ],
   "dependencies": {
+    "@babel/cli": "7.0.0-beta.44",
+    "@babel/core": "7.0.0-beta.44",
+    "@babel/plugin-proposal-export-default-from": "7.0.0-beta.44",
+    "@babel/plugin-syntax-jsx": "7.0.0-beta.44",
+    "@babel/plugin-transform-runtime": "7.0.0-beta.44",
+    "@babel/polyfill": "7.0.0-beta.44",
+    "@babel/preset-env": "7.0.0-beta.44",
+    "@babel/preset-react": "7.0.0-beta.44",
+    "@babel/preset-stage-2": "7.0.0-beta.44",
+    "@babel/runtime": "7.0.0-beta.44",
+    "@babel/types": "7.0.0-beta.44",
     "assets-webpack-plugin": "3.5.1",
     "autoprefixer": "6.3.5",
     "autosize": "3.0.15",
-    "babel-core": "6.25.0",
-    "babel-loader": "7.1.4",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-jest": "22.4.1",
+    "babel-loader": "8.0.0-beta.2",
     "babel-plugin-add-module-exports": "0.2.1",
-    "babel-plugin-syntax-jsx": "6.18.0",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-export-extensions": "6.22.0",
     "babel-plugin-transform-imports": "1.4.1",
-    "babel-plugin-transform-react-display-name": "6.25.0",
-    "babel-plugin-transform-react-jsx": "6.24.1",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-env": "1.6.0",
-    "babel-preset-stage-2": "6.24.1",
-    "babel-register": "6.24.1",
+    "babel-register": "6.26.0",
```

Comment:
This should be `@babel/register` or removed right?

Reply:
After investigation it looks like we are still using `babel-register` in a [single-location](https://github.com/Automattic/wp-calypso/blob/master/bin/build-metadata.js) which initially made me think that switching to `@babel/register` would be correct.  But after closer inspection I can't figure out why its using babel-register at all.  I'll see if i can remove it now here: https://github.com/Automattic/wp-calypso/pull/24085

Reply:
merged #24085 and removed the `babel-register`!